public/index.html

```diff
@@ -2,10 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>
-      Excalidraw | Virtual collaborative whiteboard for sketching hand-drawn
-      like diagrams
-    </title>
+    <title>Excalidraw | Hand-drawn look • Collaborative • Secure</title>
```

Comment:
```suggestion
    <title>Excalidraw | Hand-drawn looking diagrams • Collaborative • Secure</title>
```

Reply:
I'm afraid it will be already long enough again.